addon/models/time-stamp-model.js

```diff
@@ -0,0 +1,7 @@
+import Model from 'ember-data/model';
+
+export default Model.extend({
```

Comment:
Wont having this mean all the models are then 

https://guides.emberjs.com/v2.13.0/models/relationships/#toc_polymorphism? I always thought model inheritance in ember was used for specific reasons...

Reply:
Yeah, but you have to delcare a relationship that's polymorphic to get the magic as part of that. 

Otherwise it works just as single inheritance, I think. 

Hospital Run does this: https://github.com/HospitalRun/hospitalrun-frontend/blob/master/app/models/abstract.js